apps/vessel-history/features/risk-summary/risk-summary.tsx

```diff
@@ -53,11 +54,19 @@ export function RiskSummary(props: RiskSummaryProps) {
     },
     [highlightEvent, props, setMapCoordinates, viewport.zoom]
   )
+  const hasEncountersInMPAs = useMemo(() => encountersInMPA.length > 0, [encountersInMPA.length])
+  const hasEncountersInForeignEEZs = useMemo(
+    () => encountersInForeignEEZ.length > 0,
+    [encountersInForeignEEZ.length]
+  )
+  const hasFishingInMPAs = useMemo(() => fishingInMPA.length > 0, [fishingInMPA.length])
```

Comment:
Do we really need a memo for storing a boolean? 
I found [this article](https://canimerge.com/should-you-really-use-usememo-in-react-lets-find-out/) and I don't think worth it for the complexity of the operation

Reply:
Apparently, we don't need it , good catch!